src/main/java/com/example/demo/assembler/CrudAssembler.java

```diff
@@ -0,0 +1,18 @@
+package com.example.demo.assembler;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.example.demo.response.dto.UserResponseDto;
+
+public class CrudAssembler {
+	
+	@Autowired
+	UserResponseDto userResponseDto;
+	
+	public UserResponseDto toModel()
+	{
+		
+		return null;
+		
+	}
```

Comment:
This code is of no use either remove it or implement it

Reply:
Removed Class